multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -286,6 +337,65 @@ def parse_cov_dist(self):
                 if s_name in cumcov_dist_data:
                     self.add_data_source(f, s_name=s_name, section="genome_results")
 
+        # Applying the contig coverage cutoff. First, count the total coverage for
+        # every contig.
+        total_cov_per_contig = defaultdict(lambda: 0)
+        total_cov = 0
+        for s_name, perchrom in perchrom_avg_data.items():
+            for contig, cov in perchrom.items():
+                total_cov_per_contig[contig] += cov
+                total_cov += cov
```

Comment:
Can we collect this info during parsing, to avoid needing to loop through it again here?

Reply:
Yeah we can. (though as I mentioned below, doing another sequential loop is _really_ cheap, so as long as it improves readability, I prefer doing that. Though perhaps doesn't matter in this case!)